src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -267,6 +273,45 @@ class ExposureSubmissionCoordinator: NSObject, ExposureSubmissionCoordinating, R
 		)
 	}
 
+	private func createWarnOthersViewController() -> ExposureSubmissionWarnOthersViewController {
+		var vc: ExposureSubmissionWarnOthersViewController?
```

Comment:
I think this can be a let, which would resolve the force unwrap in the return.

Reply:
not because of the closure (`sink`), but this was also my first attempt.